cdap-api/src/main/java/io/cdap/cdap/api/workflow/WorkflowConfigurer.java

```diff
@@ -28,7 +30,8 @@
 /**
  * Configurer for configuring the {@link Workflow}.
  */
-public interface WorkflowConfigurer extends ProgramConfigurer, PluginConfigurer, DatasetConfigurer {
+public interface WorkflowConfigurer
+  extends ProgramConfigurer, PluginConfigurer, DatasetConfigurer, FeatureFlagsProvider {
```

Comment:
I think the `FeatureFlagsProvider` should extended by all program configurer interfaces, not just `Workflow`.